experimental/GaloisGrp/GaloisGrp.jl

```diff
@@ -12,7 +12,8 @@ import Hecke: orbit, fixed_field
 
 
 function __init__()
-  GAP.Packages.load("ferret", install = true)
+  GAP.Packages.install("ferret"; interactive=false, quiet=true)
+  GAP.Packages.load("ferret")
```

Comment:
@ThomasBreuer I guess we should support `quiet` for `load` as well? Or perhaps the "install" mode in "load" should set `quiet=true` by default?

Reply:
I am in favor of the second variant: `load` should call `install` with `quiet = true`.

With the above change, any call of `__init__` will install a new version of the package, not only in the situation when the package is not yet installed. Is this intended?

Reply:
Update of my above comment: As @fingolfin said, `GAP.Packages.install` does not install the package if the newest version is already installed. However, in order to find out what the newest version is, the function fetches the current `PackageInfo.g` file, thus one is in trouble if one is offline.

Reply:
Sure, so let's improve this in GAP.jl so that we can return to using just `load`